modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/FileEntryPicker.tsx

```diff
@@ -0,0 +1,267 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
+import {Text} from '@clayui/core';
+import ClayForm, {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import {sub} from 'frontend-js-web';
+import PropTypes from 'prop-types';
+import React, {useEffect, useRef, useState} from 'react';
+
+const FileEntryPicker = ({
+	frontendTokenDefinition,
+	frontendTokenDefinitionFileName,
+	namespace,
+}) => {
+	const inputId = `${namespace}file`;
+
+	const [feedback, setFeedbackMessage] = useState({
```

Comment:
Instead of:
```javascript
const [feedback, setFeedbackMessage] = useState({`
``` 
Should be:
```javascript
const [feedback, setFeedback] = useState({`
``` 

So we can use feedback.message